src/components/ui/Select/Select.tsx

```diff
@@ -97,7 +97,7 @@ const SelectItem = forwardRef<
     <span className="absolute left-2">
       <PrimitiveItemIndicator>
         {/* TODO:Create a new icon to use in here */}
-        <Icon icon={CaretDownIcon} size={4} />
+        <Icon icon={CheckIcon} size={4} className="text-primary" />
```

Comment:
The size of the icon itself has to be `3.5` to be `14px`. Maybe you

<img width="178" alt="image" src="https://github.com/nishiki-tech/nishiki-frontend/assets/99148565/5c4fae5c-e97d-4bd9-96b8-744e7bbca0f9">

You could add a wrapper element to make it a 16*16px square.

<img width="300" alt="image" src="https://github.com/nishiki-tech/nishiki-frontend/assets/99148565/2cd248cf-5bb5-404c-92eb-8bd287c5a4a4">
<img width="300" alt="image" src="https://github.com/nishiki-tech/nishiki-frontend/assets/99148565/a43133b9-bbb4-4d5e-8ed1-528606597b6a">